pkgs/applications/video/media-downloader/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  qt5,
+}:
+stdenv.mkDerivation rec {
+  pname = "media-downloader";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "mhogomchungu";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-EyfhomwBtdAt6HGRwnpiijm2D1LfaCAoG5qk3orDG98=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    qt5.wrapQtAppsHook
+  ];
+
+  meta = with lib; {
+    description = "A Qt/C++ GUI front end to youtube-dl";
+    homepage = "https://github.com/mhogomchungu/media-downloader";
+    license = with licenses; [ gpl2 gpl3 ];
```

Comment:
https://github.com/mhogomchungu/media-downloader/blob/main/LICENSE.txt
I think it's [gpl2Plus](https://github.com/NixOS/nixpkgs/blob/aede85ad882a1b607d1820f211580cb3a4329080/lib/licenses.nix#L432)